lib/reviews.js

```diff
@@ -80,6 +81,12 @@ class ReviewAnalyzer {
             new Review(r.state, r.publishedAt, r.url, FROM_REVIEW)
           );
           break;
+        case COMMENTED:
+          map.set(
+            login,
+            new Review(r.state, r.publishedAt, r.bodyText, FROM_REVIEW_COMMENT)
```

Comment:
I suggest we just pass `APPROVED` as the state here, like what we do for approval in normal comments. This way we don't need to call `isApprovedInComment` in `getReviewers`. We still keep the info about its source in `review.source`.

Reply:
Oh thanks, that's what I had done in the first place, forgot to reinstate it. when reverting